core/src/views/Login.vue

```diff
@@ -161,6 +161,10 @@ export default {
 		}
 	},
 
+	beforeCreate() {
+		document.title = t('core', 'Login') + ` - ${document.title}`
```

Comment:
All right! lets use an en-dash then:
```suggestion
		document.title = t('core', 'Login') + ` – ${document.title}`
```